app/views/components/gds/notificationBanner.scala.html

```diff
@@ -16,9 +16,11 @@
 
 @this()
 
-@(title: String, content: Html)
+@(title: String, content: Html, classes: Seq[String] = List.empty)
 
-<div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+@cssClasses = @{ if (classes.isEmpty) "" else s""" ${classes.mkString(" ")}""" }
```

Comment:
Is the `if` necessary? if `classes ` is an empty List would not `${classes.mkString(" ")` just produce an empty string anyway? :thinking:

Reply:
I was a bit punctilious here... In case of an empty list the html would be `<div class="govuk-notification-banner "` with a trailing blank after `govuk-notification-banner` . The `if` avoid that :smirk:

Reply:
Yes, it's necessary. We need to prefix with a blank the given classes from the `govuk-notification-banner` class.
Please note the blank before the dollar sign => `s"""<blank>$`

Reply:
Can I suggest:
`@cssClasses = @{ (classes :+ "govuk-notification-banner").mkString(" ") }`
as a simpler solution?